Middlewares/authorizations.js

```diff
@@ -0,0 +1,23 @@
+const jwt = require('jsonwebtoken');
+const { User } = require('../models');
+const { status, tokenMessages } = require('../Helpers/status&messages');
+require('dotenv').config();
+
+const JWT = process.env.JWT_SECRET;
+
+const validateToken = async (req, res, next) => {
+  const { authorization } = req.headers;
+  if (!authorization) return res.status(status.unauth).json({ message: tokenMessages.notFound });
+
+  try {
+  const { email } = jwt.verify(authorization, JWT);
+  const user = await User.findOne({ where: { email } });
+  if (!user) return res.status(status.unauth).json({ message: tokenMessages.invalid });
+  req.user = user;
+  next();
+  } catch (error) {
+    res.status(status.unauth).json({ message: tokenMessages.invalid });
```

Comment:
Oi, Heloísa, achei um res sem return aqui kk pode ser isso que esteja quebrando no avaliador. De resto n consegui identificar o erro pra n funcionar os requisitos 7 e 8
```suggestion
    return res.status(status.unauth).json({ message: tokenMessages.invalid });
```

Reply:
opa! bem observado!